includes/html/pages/addhost.inc.php

```diff
@@ -241,7 +241,14 @@
               <select name="authalgo" id="authalgo" class="form-control input-sm">
                 <option value="MD5" selected>MD5</option>
                 <option value="SHA">SHA</option>
+                <option value="SHA-224"<?=snmpv3_sha2_capable()?:" disabled"?>>SHA-224</option>
```

Comment:
This function calls a shell script (slow) would be better to store it in a temporary variable and just access that instead of calling it 4 times.